package.json

```
@@ -134,7 +135,8 @@
     "prepare": "husky install",
     "pre-commit": "lint-staged",
     "playwright:install": "playwright install --with-deps",
-    "codesandbox:install": "PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 yarn install"
+    "codesandbox:install": "PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 yarn install",
+    "deduplicate": "node scripts/deduplicate.js"
```

А почему не через --list и --fail?

```suggestion
    "deduplicate": "yarn-deduplicate --list && yarn-deduplicate --fail"
```

---

Да, ты прав, лучше через CLI команды

Копирнул решение из старого проекта)